tests/admin_filters/tests.py

```diff
@@ -737,7 +792,7 @@ def test_simplelistfilter(self):
         choices = list(filterspec.choices(changelist))
         self.assertEqual(choices[0]['display'], 'All')
         self.assertEqual(choices[0]['selected'], True)
-        self.assertEqual(choices[0]['query_string'], '?')
+        self.assertEqual(choices[0]['query_string'], '?publication-decade={0}'.format(sentinel))
```

Comment:
The problem is a bit that the Filters don't know about the defaults, they only know about the sentinel.

I can fix that if there are no defaults at all for this ModelAdmin that it will then not use the sentinels. (Because the sentinel will be None)